src/cr/cube/crunch_cube.py

```diff
@@ -200,82 +200,189 @@ def _as_array(self, include_missing=False, get_non_selected=False,
         )
         res = res + adjusted
 
+        # if prune and self.ndim < 3:
         if prune:
             return self._prune_body(res, include_transforms_for_dims)
 
         return res
 
+    def _prune_3d_body(self, res, transforms):
+        mask = np.zeros(res.shape)
+        for i, prune_inds in enumerate(self.prune_indices(transforms)):
+            rows_pruned = prune_inds[0]
+            cols_pruned = prune_inds[1]
```

Comment:
Changing this isn't a requirement, but I find it a bit confusing that rows_pruned and cols_pruned don't match the numpy rows/cols indexes. Switching back and forth between which is first and which is second takes a little bit of mental gymnastics.

Reply:
I'm not sure I get this. Let's :telephone_receiver: about it :smiley_cat: